source/_cookie_banner.html.erb

```diff
@@ -0,0 +1,21 @@
+<div id="cookie-banner" class="govuk-cookie-banner" hidden="true" data-nosnippet role="region" aria-label="Cookies on <%= config[:service_name] %>" data-module="cookie-banner">
+  <div class="govuk-cookie-banner__message govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h2 class="govuk-cookie-banner__heading govuk-heading-m">Cookies on <%= config[:service_name] %></h2>
+        <div class="govuk-cookie-banner__content">
+          <p class="govuk-body">We’d like to use analytics cookies so we can understand how you use this website and make improvements.</p>
+        </div>
+      </div>
+    </div>
+    <div class="govuk-button-group">
+      <button type="button" class="govuk-button" data-module="govuk-button" data-function="accept-cookies">
+        Accept analytics cookies
+      </button>
+      <button type="button" class="govuk-button" data-module="govuk-button" data-function="reject-cookies">
+        Reject analytics cookies
+      </button>
+      <%= link_to 'How we use cookies', '/cookies', class: 'govuk-link' %>
```

Comment:
Small thing: don't think you need to add the `govuk-link` here since we've got [`$govuk-global-styles: true;`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-global-styles) in our stylesheet:

```suggestion
      <%= link_to 'How we use cookies', '/cookies' %>
```